### PR TITLE
Make gcloud bucket configurable, enables Flank for free Firebase plan.

### LINKED ISSUE #53: What is the reason to require paid Firebase plan?
TL;TR: we want *sometimes* run our instrumentation tests in Firebase, but most of the time run them on internal emulators pool.

So we're completely fine with free plan quotas but Flank requires paid plan (for Google Cloud Storage bucket I guess).

What is the reason to require paid plan and can this be fixed to allow test runs on free plan? Thanks!

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Fixes #53. 

This PR allows user to set gcloud-bucket from free Firebase account and avoid paid plan if it's not required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ fetch-xml-files: If the result xml files should be fetched
 fetch-bucket: If the bucket containing logs and artifacts should be fetched
 gcloud-path: The path to the glcoud binary
 gsutil-path: The path to the gsutil binary
-
+gcloud-bucket: The Google Cloud Storage bucket to put/pull files during test run
 ```
```
Comment (shawnshaligram) [inline README.md]: can you also update the part where one of the pre-requisites mentions having a paid subscription

Comment (artem-zinnatullin) [inline README.md]: Done, thanks!

**Comment by artem-zinnatullin** [comment]
@renas PTAL

**Comment by renas** [comment]
On PTO this week @artem-zinnatullin will check it next week when I'm back.

**Comment by artem-zinnatullin** [comment]
Alright, have good relaxing days :)

Thanks for letting us know!

**Comment by renas** [comment]
Thanks for contributing.

**Pull request merged**